resources/build/set-bundle-versions-and-settings.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

We should be using `env` throughout to signify that we want the bash version from the path, even though we are attempting to run the scripts with the correct version of bash.